bindings/kotlin/README.md

```diff
@@ -0,0 +1,24 @@
+# Matrix rust components kotlin
+
+This project and build scripts demonstrate how to create an aar and how to import it in your android projects.
+
+## Prerequisites
+
+* the Rust toolchain
+* UniFFI - `cargo install uniffi_bindgen`
```

Comment:
we have recently deprecated the sh from apple and moved all into an xtask job - an important reason being that we don't have uniffi_bindgen mismatches anymore. Could we maybe do the same here? I'd rather not add bash-scripts again :( ...

Reply:
Can we do that later?